recipe/meta.yaml

```diff
@@ -17,7 +17,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
+  skip: True  # [unix and not py35]
```

Comment:
There was an error in the last statement. Python is used for building the test suite and hence used in the build requirement.